packages/shared/features/onboardingFeatures.ts

```diff
@@ -38,7 +38,7 @@ const onboardingFeatures = {
                     enabled: false,
                 },
                 ledgerBackup: {
-                    enabled: false,
+                    enabled: true,
```

Comment:
I think we're supposed to leave the IOTA-based feature flags set to false until V2 can support it